### PR TITLE
Update ops guide to minimize certain references

### PR DESCRIPTION
**Pull request by triclambert**
Minimize references to "framework" and "SDK".

### CODE REVIEW

File: docs/pages/operations-guide.md [hunk 1]
```diff
@@ -797,7 +797,7 @@ If you are running DC/OS 1.9 or older, or a version of the service that is older
 
 1. Stop the service. From the DC/OS CLI, enter `dcos package uninstall --app-id=<instancename> <packagename>`.
    For example, `dcos package uninstall --app-id=kafka-dev confluent-kafka`.
-1. Clean up remaining reserved resources with the framework cleaner script, `janitor.py`. See [DC/OS documentation](https://docs.mesosphere.com/1.9/deploying-services/uninstall/#framework-cleaner) for more information about the framework cleaner script.
+1. Clean up remaining reserved resources with the `janitor.py` script. See [DC/OS documentation](https://docs.mesosphere.com/1.10/deploying-services/uninstall/#framework-cleaner) for more information about the cleaner script.
```
Comment (elezar) [inline docs/pages/operations-guide.md]: I thought we didn't want users to run janitor on 1.10 clusters?

Comment (benclarkwood) [reply docs/pages/operations-guide.md]: Yeah, super dont

Comment (mpereira) [reply docs/pages/operations-guide.md]: This section seems to be for older versions though?

> Older versions
> If you are running DC/OS 1.9 or older, or a version of the service that is older than 2.0.0-x, follow these steps:
> 1. Stop the service.
> ...

Comment (triclambert) [reply docs/pages/operations-guide.md]: That is correct, @mpereira—this is the section for DC/OS versions older than 1.10.

Comment (triclambert) [reply docs/pages/operations-guide.md]: There may, of course, be other stale content in the doc.

**Comment by gabrielhartmann** [comment]
@triclambert:  Going through our long PR list and cleaning up.  Is this ready to merge?

File: docs/pages/operations-guide.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ redirect_from: "/ops-guide"
 
 <!-- {% raw %} disable mustache templating in this file: retain templated examples as-is -->
 
-This operations guide describes how to manage stateful DC/OS services that are based on the DC/OS SDK. Consult the [Developer Guide](developer-guide.html) to learn how to build DC/OS SDK services.
+This operations guide describes how to manage stateful DC/OS services that are based on the DC/OS SDK. The SDK, while referenced throughout this document, is not yet generally available.
```
Comment (triclambert) [inline docs/pages/operations-guide.md]: @ToddGreenstein are you ok with this caveat?  Can it be expanded somehow?

**Comment by nickbp** [comment]
ping

**Pull request merged**